### PR TITLE
tactile_paving removed

### PR DESCRIPTION
**Pull request by jbgriesner**
This PR adapts the accessibility block to the following specifications:

* if wheelchair and toilets:wheelchair are both undefined : no accessibility block
* if wheelchair and toilets:wheelchair are both defined with unknown values: no accessibility block

Simple tests (from the specifications) come along the changes.

### CODE REVIEW

File: idunn/blocks/services_and_information.py [hunk 1]
```diff
@@ -23,9 +22,14 @@ def from_es(cls, es_poi, lang):
...
 
         raw_wheelchair = properties.get("wheelchair")
-        raw_tactile_paving = properties.get("tactile_paving")
         raw_toilets_wheelchair = properties.get("toilets:wheelchair")
 
+        if all(
+            s is None
+            for s in (raw_wheelchair, raw_toilets_wheelchair)
+        ):
+            return None
```
Comment (amatissart) [inline idunn/blocks/services_and_information.py]: The check is redundant and can be removed.

File: tests/test_accessibility.py [hunk 2]
```diff
@@ -25,9 +25,68 @@ def test_accessibility_unknown():
...
+    assert resp['name'] == "Boulangerie Patisserie Peron"
+    assert resp['local_name'] == "Boulangerie Patisserie Peron"
+    assert resp['class_name'] == 'bakery'
+    assert resp['subclass_name'] == 'bakery'
+    assert resp['blocks'] == []
+
+def test_wheelchair(cinema_multiplexe):
+    """
+    Test that when wheelchair is defined but not toilets_wheelchair
+    we still have a correct block
```
Comment (amatissart) [inline tests/test_accessibility.py]: this description does not match the test, where both values are "yes"

File: idunn/blocks/services_and_information.py (hunk 1 above)
Comment (jbgriesner) [reply idunn/blocks/services_and_information.py]: your are right: removed

File: tests/test_accessibility.py (hunk 2 above)
Comment (jbgriesner) [reply tests/test_accessibility.py]: Indeed: it has been fixed

**Pull request merged**